README.md

```diff
@@ -1,18 +1,17 @@
-# nRF Connect SDK App Index
-
-An index of applications built for the nRF Connect SDK.
+# nRF Connect SDK Add-on index
 
+This repository includes a collection of index pages for publicly available nRF Connect SDK Add-ons, which are supplementary modules that extend the functionality of the [nRF Connect SDK](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/index.html). You can access the index from the [nRF Connect for VS Code extension](https://docs.nordicsemi.com/bundle/nrf-connect-vscode/page/index.html).
```

Comment:
```suggestion
This repository includes a collection of index pages for publicly available nRF Connect SDK Add-ons, which are supplementary modules that extend the functionality of the [nRF Connect SDK](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/index.html). You can access the index from the [nRF Connect for VS Code extension](https://docs.nordicsemi.com/bundle/nrf-connect-vscode/page/index.html), and browse it on [its webpage](https://nrfconnect.github.io/ncs-app-index/)
```
My point is to mention both ways it can be accessed, the url will most probably change overtime.